modules/src/error_msg.py

```diff
@@ -0,0 +1,55 @@
+
```

Comment:
Unsure, so asking for clarification:
Split into two files, one for error messages (error_msg.py) and one for examples (examples.py)?
I understand storing them a directory above, to use for both tests and src files.